qualtran/_infra/controlled.py

```diff
@@ -462,3 +462,20 @@ def as_cirq_op(
             sub_op.controlled_by(*ctrl_qubits, control_values=self.ctrl_spec.to_cirq_cv()),
             cirq_quregs | ctrl_regs,
         )
+
+    def _circuit_diagram_info_(self, args: cirq.CircuitDiagramInfoArgs) -> cirq.CircuitDiagramInfo:
+        """Default diagram info that uses register names to name the boxes in multi-qubit gates.
+
+        Descendants can override this method with more meaningful circuit diagram information.
+        """
+        info = cirq.circuit_diagram_info(self.subbloq, default=None)
+
+        if info is None:
+            return super()._circuit_diagram_info_(args)
+
+        wire_symbols = []
+        for cv in self.ctrl_spec.cvs:
+            for bit in cv.ravel():
+                wire_symbols.append('@' if bit else '@(0)')
+        wire_symbols += info.wire_symbols
+        return cirq.CircuitDiagramInfo(wire_symbols=wire_symbols)
```

Comment:
Add tests for this?

Reply:
turns out this is circular, reverted the change in GWR and instead only calling this in Controlled

Reply:
added tests